cernopendata/modules/fixtures/data/docs/cod-categories-simulated-datasets/cod-categories-simulated-datasets.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "body": {
+      "content": "cod-categories-simulated-datasets.md",
+      "format": "md"
+    },
+    "short_description": {
+      "content": "Description of the categories for the simulated datasets."
+    },
+    "slug": "cod-categories-simulated-datasets",
```

Comment:
The `slug` had better not start with "cod-" because it would appear in the URL and people wouldn't know what it is...  Please use something fully readable like `simulated-dataset-categories` which will make nicer URLs.